### PR TITLE
Rename TryAgainLater status code to ServerTimeout

### PR DESCRIPTION
**Pull request by smarterclayton**
Follow up from #4263

### CODE REVIEW

File: pkg/api/rest/resttest/resttest.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func (t *Tester) TestCreateGeneratesNameReturnsTryAgain(valid runtime.Object) {
 	objectMeta.GenerateName = "test-"
```
Comment (saad-ali) [inline pkg/api/rest/resttest/resttest.go]: Rename this method

**Comment by saad-ali** [comment]
LGTM mod nit

**Comment by smarterclayton** [comment]
Fixed

----- Original Message -----

**Comment by saad-ali** [comment]
LGTM, merging.

**Pull request merged**